src/site/content/en/secure/what-is-mixed-content/index.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: post
+title: What is mixed content?
+authors:
+  - johyphenel
+  - rachelandrew
+date: 2019-09-07
+updated: 2020-09-16
+description: >
+  Mixed content occurs when initial HTML is loaded over a secure HTTPS connection,
+  but other resources are loaded over an insecure HTTP connection.
+tags:
+  - security
+---
```

Comment:
```suggestion
tags:
  - security
  - network
  - privacy
  - html
  - css
  - javascript
  - images
  - media
---
```
The last 6 tags are a bit of a stretch but I suppose it's good to be more comprehensive with our tagging approach (because mixed content definitely can affect all those areas)

Reply:
I'll add these to the other one as well